decidim-core/spec/services/decidim/open_data_exporter_spec.rb

```diff
@@ -93,6 +93,54 @@
       it_behaves_like "open data exporter"
     end
 
+    describe "with moderations" do
```

Comment:
Can you add the check for both `resource_title`s in https://github.com/decidim/decidim/blob/cd392e9eeed1d8b14de4a7245fc3db91dfdf8320/decidim-core/spec/services/decidim/open_data_exporter_spec.rb#L183 ?